client/css/base/_typography.css

```diff
@@ -1,3 +1,50 @@
 /**
  * Basic typography style for copy text
  */
+
+a {
+  font-weight: bold;
+  color: #fff;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+hr {
+  border: 0;
+  border-bottom: 1px dashed $dark-grey;
+  width: calc(100% - 40px);
+}
+
+blockquote {
+  padding-left: 20%;
+  padding-right: 10px;
+  background-color: rgba(0, 0, 0, .2);
+}
+
+blockquote cite {
+  padding-left: 20px;
+  padding-right: 10px;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+  display: inline-block;
+}
+
+pre {
+  background-color: rgba(0, 0, 0, 0.1);
+  padding: 1rem;
+  color: $light-brand-color;
+  width: 100%;
+}
+
+code {
+  background-color: rgba(0, 0, 0, 0.2);
```

Comment:
Maybe there is a reason why colors with opacity cannot be stored in a variable - if there is, let me know :) If it can be stored in a variable, I'd use variables everywhere